### DISCUSSION TITLE
Error when compiling a shared BEAM library

### DISCUSSION THREAD

**Discussion by zw-mphys** [nrc-cnrc/EGSnrc#1186] [Questions] [answered]
Hello! I am attempting to compile an accelerator as a shared library. I have already built my beam and when I input: "make library" I get an error: 

`  generated mortjob.mortran
  fortran output: libBEAM_mybeam_osx.F
  mortlst output: libBEAM_mybeam_osx.mortlst

  Mortran compiling ...  OK
gfortran -c -fPIC -O2 -mtune=native  -DCOMPILE_TIME="\"2024-09-12 23:47:03 UTC\"" -DGIT_HASH="\"fe83057\"" -o libBEAM_mybeam_osx.o libBEAM_mybeam_osx.F
libBEAM_mybeam_osx.F:47223:17:

47223 |       write(i_log,26170)luxury_level,jseed
      |                 1
Error: Symbol 'i_log' at (1) has no IMPLICIT type
make: *** [libBEAM_mybeam_osx.o] Error 1
`

I am running EGS on an M1 Mac (macOS Sonoma 14.5). Are there any ideas on how to resolve this?

**Comment by rtownson** [comment]
Can you post the installation log from `hen_house/log`? I assume that `make` alone also fails with the same error? Does it also fail with one of the example accelerators, e.g. `EX16MVp`?

**Reply by zw-mphys** [reply, marked as answer]
Hi! Sorry for the late response. I was able to fix this by reinstalling EGS
